Database migration not running in stable

I just saw that MapKnitter stable test server is failing every time; thought it was related to version numbers in #862 but i see error:

```

[1Bdocker-compose exec -T web bash -l -c "sleep 10 && rake db:migrate && rake assets:precompile"
mesg: ttyname failed: Inappropriate ioctl for device
rake aborted!
Bundler::GemNotFound: Could not find request_store-1.4.1 in any of the sources
/app/config/boot.rb:3:in `<top (required)>'
/app/config/application.rb:1:in `<top (required)>'
/app/Rakefile:4:in `<top (required)>'
/usr/local/bundle/gems/rake-12.3.3/exe/rake:27:in `<top (required)>'
(See full trace by running task with --trace)
Makefile:16: recipe for target 'redeploy-container' failed
make: *** [redeploy-container] Error 1
Build step 'Execute shell' marked build as failure
Finished: FAILURE
```

https://jenkins.laboratoriopublico.org/job/Mapknitter-Stable/239/console

I confirmed that `schema.rb` is not updated, so `rake db:migrate` has not run; but it seems like the first command in the sequence `docker-compose exec -T web bash -l -c "sleep 10 && rake db:migrate && rake assets:precompile"` -- ...hmm...

Looking deeper now.

Aha, the only build of this on unstable faced the same error:

```

[1Bdocker-compose exec -T web bash -l -c "sleep 10 && rake db:migrate && rake assets:precompile"
mesg: ttyname failed: Inappropriate ioctl for device
rake aborted!
Bundler::GemNotFound: Could not find request_store-1.4.1 in any of the sources
/app/config/boot.rb:3:in `<top (required)>'
/app/config/application.rb:1:in `<top (required)>'
/app/Rakefile:4:in `<top (required)>'
/usr/local/bundle/gems/rake-12.3.3/exe/rake:27:in `<top (required)>'
(See full trace by running task with --trace)
Makefile:16: recipe for target 'redeploy-container' failed
make: *** [redeploy-container] Error 1
Build step 'Execute shell' marked build as failure
Finished: FAILURE
```

https://jenkins.laboratoriopublico.org/job/Mapknitter-Unstable/196/console

--- a/config/initializers/paper_trail.rb
+++ b/config/initializers/paper_trail.rb
@@ -1,2 +1,1 @@
-PaperTrail.config.track_associations = false
 PaperTrail.serializer = PaperTrail::Serializers::JSON

